README.md

```diff
@@ -1,23 +1,36 @@
 # EthicML
 
-EthicML exists to solve the problems we've found with off-the-shelf fairness comparison packages.
+[![Checked with mypy](http://www.mypy-lang.org/static/mypy_badge.svg)](http://mypy-lang.org/)
+![](https://github.com/predictive-analytics-lab/EthicML/workflows/EthicML%20CI/badge.svg)
 
-These other packages are useful, but given that we primarily do research,
+
+
+EthicML is the premier library for performing and assessing __algorithmic fairness__.
+Unlike other libraries, EthicML isn't an education tool, but rather a __researcher__'s toolkit.
```

Comment:
btw, did you mean to make these terms bold? or were you confused by Roam's syntax and wanted to make them italic?

Reply:
well, I started off my wanting to make them italic, then they came out bold and I didn't mind it

Reply:
I see